AbletonMCP_Remote_Script/__init__.py

```
@@ -779,12 +779,21 @@ def _find_browser_item_by_uri(self, browser_or_item, uri, max_depth=10, current_
                     browser_or_item.midi_effects
                 ]
-                
+                # Also walk plugins / max_for_live / user_library / packs / samples
+                # so URIs from get_browser_items_at_path for plugins actually load.
+                # — lukegimza, 2026-05-14
+                for extra_attr in ('plugins', 'max_for_live', 'user_library', 'packs', 'samples'):
+                    if hasattr(browser_or_item, extra_attr):
+                        try:
+                            categories.append(getattr(browser_or_item, extra_attr))
+                        except Exception:
+                            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)